markdownfmt/testfiles/lists.output.md

```diff
@@ -85,6 +84,7 @@ Nested:
 4. more nested
    - coffee
      - beer
+
```

Comment:
why this line here?

Reply:
No good reason (: Wil adjust

Reply:
Actually there is: input specifies new line so there is one (checked by `isHardNewLine`) so I would keep it (: Any rule against it? ;p